db/migrate/20170619161514_add_options_to_ext_management_system.rb

```diff
@@ -0,0 +1,32 @@
+class AddOptionsToExtManagementSystem < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options
+  end
+  class CustomAttribute < ActiveRecord::Base
+    serialize :serialized_value
+  end
+
+  def up
+    add_column :ext_management_systems, :options, :text
+
+    say_with_time("Migrating Kubernetes provider options") do
+      ExtManagementSystem.where(:type => "ManageIQ::Providers::Kubernetes::ContainerManager").each do |cp|
+        migrated_options = {}
+        CustomAttribute.where(
+          :resource_type => cp.type,
+          :resource_id   => cp.id,
+          :section       => ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job::ATTRIBUTE_SECTION,
+          :name          => ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job::PROXY_ENV_VARIABLES
```

Comment:
@enoodle as mentioned before I am not sure if we can afford this dependency to the Kubernetes module. cc @blomquisg You may have to hard-code the values here.